docs/react-v9/contributing/dev-workflow.md

```diff
@@ -21,20 +21,22 @@ We don't have an official branch naming policy. Since every contribution happens
 
 The inner development loop usually involves these steps:
 
-First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages. This can take some time.
+First, `cd` to the root folder of the repo (usually `fluentui`) and run `yarn` to install dependencies and link packages. This can take some time if it's the first time you've run it.
```

Comment:
Generally I would keep these changes in a separate PR, but we can leave them since they are small.